cmd/sbrmi/src/lib.rs

```diff
@@ -212,7 +212,7 @@ fn cpuid(
     }
     #[derive(Clone)]
     struct WorkspaceRef<'a, 'b> {
-        cell: Arc<RefCell<SbrmiWorkspace<'a, 'b>>>,
+        cell: Rc<RefCell<SbrmiWorkspace<'a, 'b>>>,
```

Comment:
I originally just silenced [the clippy warning](https://rust-lang.github.io/rust-clippy/master/index.html#/arc_with_non_send_sync), but on reading the description of the lint decided to try replacing `Arc` with `Rc`. That compiles, the tests pass, and clippy is happy. But I do not understand what this code is doing or why, so if someone that does could please double-check that this is acceptable, I'd appreciate it.

Reply:
I think this is okay but this is probably a @bcantrill question

Reply:
This looks fine to me. I think @mkeeter put this in to replace some unsafe hackery once the `raw-cpuid` crate added some functionality, and there's no `unsafe` left so it's fine to rely on clippy. It (clippy) is certainly correct that `Arc<RefCell<_>>` is very suspicious, since `Arc` is for sharing with multiple threads and `RefCell` is not threadsafe.